its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/BaseDirTest.java

```diff
@@ -149,8 +148,7 @@ void projectBaseDir_Relative() {
     // projectBaseDir = "..\.." is relative to the projectDir. That is "C:\Windows\Temp\", so component keys should start with tempDirectoryName.
     var tempDirectoryName = context.projectDir.getParent().getFileName().toString();
     context.begin
-      .setProperty("sonar.projectBaseDir", Paths.get("..", "..").toString())  // Relative from scanner working directory
-      .setProperty("sonar.scanner.useSonarScannerCLI", "true");   // TODO: remove in SCAN4NET-861
+      .setProperty("sonar.projectBaseDir", Paths.get("..", "..").toString());  // Relative from scanner working directory
```

Comment:
When we set this property, the `sonar.projectBaseDir` will be added twice to the properties file:
* Once as directoy.Fullname (fully qualified) because the project base directory get's calculated
* Later it is added because it was a user supplied command line argument

In the scannerEngineInput we do the same, but instead of suppling it twice, we override the fully qualified directory with the value from command line argument. The result is the same, though.